semantics/language/common/syntax.k

```diff
@@ -4,10 +4,11 @@ module C-SYNTAX
      imports INT-SYNTAX
      imports BOOL-SYNTAX
      imports LIST
-     imports KORE-COMPAT
 
      imports C-DYNAMIC-SYNTAX
 
+     syntax KBott ::= "(" K ")" [bracket]
```

Comment:
in order to avoid changing hundreds and hundreds of lines of code in the C semantics, I introduced legacy brackets with this production so that users don't have to say ``` `` ``` for brackets.